.travis.yml

```diff
@@ -4,4 +4,6 @@ rust:
 
 cache: cargo
 script:
-  - cargo build -p rstar-benches && cargo test --all-features -p rstar
\ No newline at end of file
+  - rustup component add rustfmt --toolchain stable-x86_64-unknown-linux-gnu
```

Comment:
Is it wise to specify the toolchain here? Doesn't this break using the build matrix if we decide to?